app/src/main/res/layout/topic_summary_view.xml

```diff
@@ -10,22 +11,19 @@
   </data>
 
   <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="140dp"
-    android:layout_height="160dp"
-    android:layout_margin="10dp"
+    android:layout_width="match_parent"
+    android:layout_height="168dp"
```

Comment:
As per the mock in your description this height should be `160dp` but also I am suspecting that reducing the height will make the card look much more like square which we do not want. Maybe be checking this with @mschanteltc would be a good idea.

Reply:
Also, if you are giving height to every child then it can be "wrap_content" too.

Reply:
I tried `wrap_content` but image doesn't look proper so gave fixed height to all child imageviews.

Regarding height setting it to 168dp is giving potrait layout, otherwise yes as you said its showing square.  

@mschanteltc  is it good to go with 168dp rather than 160dp?

Reply:
What is the reason for increasing the height? Is it so that the thumbnail image can be "zoomed out" more?

Reply:
Not to zoom out the image but, if the screen is wide it looks square or in landscape so adjusted height to show it portrait. I have changed it to 160 dp. This is the screenshot PTAL

![Screenshot_20191112-105355](https://user-images.githubusercontent.com/16301028/68644591-7c02f780-053b-11ea-85e8-a50a1a2a2b9a.png)